schema/schema.go

```diff
@@ -371,7 +392,12 @@ func fromCollectionModel(model interface{}) (*Schema, error) {
 	}
 
 	if len(pk) == 0 {
-		return nil, fmt.Errorf("no primary key defined in schema")
+		if !embModel {
+			return nil, fmt.Errorf("no primary key defined in schema")
+		}
+
+		sch.PrimaryKey = append(sch.PrimaryKey, "id")
```

Comment:
should we call it `_id`? As user may have `id` field and then `_` prefix represents internal field?